packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -86,12 +87,124 @@ const [phpLoaderModule, wpLoaderModule] = await Promise.all([
 ]);
 monitor.setModules([phpLoaderModule, wpLoaderModule]);
 php.initializeRuntime(
-	await loadPHPRuntime(phpLoaderModule, monitor.getEmscriptenArgs(), [
-		wpLoaderModule,
-	])
+  await loadPHPRuntime(phpLoaderModule, {
+    ...monitor.getEmscriptenArgs(), 'print': function (text) {
+      console.log('stdout: ' + text)
+    },
+    'printErr': function (text) {
+      console.log('stderr: ' + text)
+    }
+  }, [
+    wpLoaderModule,
+  ])
 );
 patchWordPress(php, scopedSiteUrl);
 
+
+php.writeFile(
+  '/wordpress/tests/phpunit/includes/bootstrap.php',
+  php.readFileAsText('/wordpress/tests/phpunit/includes/bootstrap.php').replace("'gd',", '')
+);
+
+php.writeFile(
```

Comment:
The build process "minifies" PHP files. This overrides `wp-config.php` to add back newlines and comments so that WP-CLI detects the WP setup correctly. Ideally this should be done properly in the build process.

Reply:
The minification happens here:

https://github.com/WordPress/wordpress-playground/blob/da38192af57a95699d8731c855b82ac0222df61b/packages/playground/compile-wordpress/Dockerfile#L146-L153

`-not -name 'wp-config.php' ` should do the trick